staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go

```diff
@@ -110,15 +110,15 @@ func ConnectResource(connecter rest.Connecter, scope RequestScope, admit admissi
 			scope.err(err, w, req)
 			return
 		}
-		if admit.Handles(admission.Connect) {
+		// TODO: what about validation admission here?
```

Comment:
Does it make sense to run revising webhooks on proxy connections? The body isn't a thing that can be revised, it's a stream? I'd expect only validating webhooks to run on these.

Reply:
For the time being we have to call both. I added a TODO here (as we had already in the delete handler). I would like to keep any admission plugin refactoring out of this PR. When we changed all of them to validation, we can fix those TODOs.